src/attributes.jl

```diff
@@ -920,8 +920,12 @@ most recent call to [`compute_conflict!`](@ref).
 Possible values are:
 * `COMPUTE_CONFLICT_NOT_CALLED`: the function [`compute_conflict!`](@ref) has
   not yet been called
-* `NO_CONFLICT_EXISTS`: there is no conflict because the problem is feasible
-* `NO_CONFLICT_FOUND`: the solver could not find a conflict
+* `NO_CONFLICT_EXISTS`: there is no conflict available, because the solver could 
+  prove that the problem is feasible (i.e. there are no conflicts)
+* `NO_CONFLICT_FOUND`: there is no conflict available. There may be several
+  reasons for this: the function [`compute_conflict!`](@ref) has finished
+  and decided that the problem is feasible, the function has failed to 
```

Comment:
If it decided that the problem is feasible then it should return the other status. What could hapoen is that the problem is feasible but the solver did not see that, it just did not found anything but does not know whether itvis because the problem is feasible or it just did not search hard enough

Reply:
@odow what do you think of this?